tests/consensus_tests/test_named_vector_crud.py

```
@@ -73,7 +73,7 @@ def test_create_vector_no_optimization(tmp_path: pathlib.Path):
     VECTOR_DIM2 = 99
     assert_project_root()
 
-    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS, port_seed=10000)
+    peer_api_uris, peer_dirs, bootstrap_uri = start_cluster(tmp_path, N_PEERS)
```

Ruff (0.15.10)

[warning] Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)